app/src/main/kotlin/com/intive/picover/parties/model/Party.kt

```diff
@@ -4,4 +4,6 @@ data class Party(
 	val id: Int,
 	val title: String,
 	val description: String,
-)
+) {
+	constructor() : this(0, "", "")
```

Comment:
there is an error, when using object without default values, so used this approach. Alternative is use: val id: Int = 0 etc

Reply:
The error is:
com.google.firebase.database.DatabaseException: Class com.intive.picover.parties.model.Party does not define a no-argument constructor.

I see another interesting solution for that -> use custom annotation class together with usage of No-arg compiler plugin ( https://kotlinlang.org/docs/no-arg-plugin.html)

I’ve done some quick tests and looks like it works (just annotate the data class with custom anotation and no need to use no-argument constructor or default params)

Reply:
Not at all :D i left this intentionally to stimulate some of team members to discuss this, as our review is quite dead :D I've added proposal that i mentioned on one of guild meetings -> remote model and UI model